src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt

```diff
@@ -341,6 +346,7 @@ abstract class DatabaseTest {
   protected val organizationsDao: OrganizationsDao by lazyDao()
   protected val organizationUsersDao: OrganizationUsersDao by lazyDao()
   protected val plantingsDao: PlantingsDao by lazyDao()
+  protected val plantingSeasonsDao: PlantingSeasonsDao by lazyDao()
```

Comment:
Do we need a test to check that only one active planting season can be present?

Reply:
I have mixed feelings about this. On the one hand, it's a behavior of the system, even if it's only a second layer of defense against something the app should never try to do. So of course it should be verified by a test that will catch someone messing up the constraint by accident in the future.

On the other hand, the exact same thing is true of the foreign-key constraint on the planting site ID column. Verifying that the database doesn't let us refer to a nonexistent planting site seems (to me) not worthwhile, especially if we apply that thinking consistently and add tests to cover _all_ the integrity constraints in the database.

Maybe the argument in favor of testing this but not the foreign key is that an exclude constraint is more esoteric than a foreign key, and the test serves as documentation for people who aren't as familiar with the syntax? Or maybe the deciding factor is that the constraint doesn't apply to all rows?

Reply:
I've added a test for this, even if only so we have something more concrete to discuss.

Reply:
Thanks for sharing more context. Wasn't too sure on expectation.
Makes sense to not require tests for these cases going forward.
Would you prefer to revert the added test or keep it for discussion purposes?

Reply:
I'll revert it. Thanks for the discussion, though -- definitely worth raising as a question.

Reply:
Thanks, Steve. Sounds good 👍